### ISSUE TITLE
Input - "Event editing fields to show" some fields are not by default checked.

### ISSUE THREAD

**Issue by paaljoachim** [joedolson/my-calendar#223]
I deleted the plugin and all the data and uploaded the newest version.

Upon creating a new event I did not have an option to add a new location directly in the Edit screen. I went to Settings -> Input and noticed that by default that Excerpt, Host, Location Fields, Registration Settings, Special Scheduling Options and "Administrators see all input options" are all unchecked.

![Screenshot 2021-09-21 at 01 31 58](https://user-images.githubusercontent.com/5323259/134089944-220d232b-6bd1-48e1-aa02-8e935d479ef4.jpg)

I would suggest having all of the above fields be prechecked by default except "Administrators see all input options."

As it would make it easier in creating a new event just as one has begun to use the plugin.

**Comment by joedolson** [comment]
I definitely do *not* want all options checked by default. Location fields is only available as an option for historical reasons - because it used to be the only option, and some users will be accustomed to that. I would prefer to actively discourage using it. Other settings are just rarely used; I want to offer a starting view that isn't super complicated, but offers the most commonly needed settings. Things like registration settings, special scheduling options, and host are all quite rarely used. 

I can see an argument for adding the Excerpt field to default settings; but mostly I'd rather not.

**Comment by paaljoachim** [comment]
I see your point.

An alternative.
I am wondering if we should find a way to give a hint when a user adds a new event that the Input fields can be controlled from the Settings > Input screen.

**Comment by joedolson** [comment]
I could implement an admin pointer pointed to the screen options. Most users should only be directed to screen options, though - any user who can access the add event screen is able to access screen options, but only admins or users with My Calendar settings permissions can access the site settings.

**Comment by joedolson** [comment]
As I work on this, I'm now thinking that I'll reduce this to a single option, and always have both fields & selector grouped together. With the toggle to hide the field by default, there's no pressing reason not to have both, and it allows me to do a lot of code simplifications...

**Comment by paaljoachim** [comment]
I just tested.
Adding a new event as well as going to an existing event.
Opening Screen Options. Unchecked Registration Settings and Repetition Pattern.
Clicked Apply.

I would assume that Registration Settings and Repetition Pattern would be hidden from Add event and current event screens. But both are still seen....hmmm...

I then went to Settings -> Input to see if these fields have been automatically unchecked there, but they were not.

**Comment by joedolson** [comment]
In https://github.com/joedolson/my-calendar/commit/fe3f069b6b612eb29d2e738c98ee4979397b3801, I changed to all fields turned on by default. My primary concern about that was the volume of fields that are turned on with locations, if that's enabled, but with the revised location fields section, that's a significantly lower concern. 

Closing this. 

Balancing input management between admin settings & screen options is a separate issue.
